www/js/userInfoService.js

```diff
@@ -0,0 +1,57 @@
+var OFFLINE_ONLY = true;
```

Comment:
This is fine for now. But we should create a .gitignore'd config file that specifies whether to run in demo or production mode.